src/lib.rs

```diff
@@ -33,6 +33,9 @@ pub mod syscalls;
 
 pub use libtock_codegen::main;
 
-// Dummy structure to force importing the panic_handler and other no_std elements when nothing else
-// is imported.
+pub(crate) mod drivers;
+pub use drivers::*;
+
+/// Dummy structure to force importing the panic_handler and other no_std elements when nothing else
+/// is imported.
 pub struct LibTock;
```

Comment:
Can this be removed?

Reply:
I'm not sure. I haven't verified that it is necessary. But according to the comment, no.

Reply:
I believe that `extern crate libtock` would work as well.